pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -91,12 +87,30 @@ func validateClusterServiceBrokerSpec(spec *sc.ClusterServiceBrokerSpec, fldPath
 		}
 	}
 
+	commonErrs := validateCommonServiceBrokerSpec(&spec.CommonServiceBrokerSpec, fldPath)
```

Comment:
why create a var and an if? I think append works fine on empty slices

Reply:
I think the assignment and if can be combined in one line if we keep this format.

Reply:
true.